app/DTO/UpdateUserDto.php

```diff
@@ -9,6 +9,7 @@ class UpdateUserDto
     public function __construct(
         public ?string $firstName,
         public ?string $lastName,
+        public ?string $avatar_url,
```

Comment:
Please be consistent with naming - use camelCase or snake_case not some mix of them